AUTHORS.md

```diff
@@ -9,13 +9,13 @@ Authors
 * [Aidin Gharibnavaz](https://github.com/aidin36)
 * [AJ ONeal](https://github.com/coolaj86)
 * [Alcaro](https://github.com/Alcaro)
-* [Alexander Mankuta](https://github.com/pointlessone)
```

Comment:
So it not exactly alphabetically sorted ;). I think we should punt this to another PR just to keep things clean. 

And perhaps we will want to do something in that PR in the automated test suite to ensure that the file is actually sorted.

Reply:
Agree, maybe we should enforce a structure of this file ? Like having everyone use a certain format: 
`[NAME](github_url)`
provide as you suggest a test to verify order/structure and a script that can fix the failing test.

Reply:
Sure, I filed #8543 to track this. Let's put that aside and only add your name in (approximately) the right spot.

I'm otherwise ready to approve this.

Reply:
Cool,  I have reverted the changes on `AUTHORS.md` and pushed my name in the right spot.